languages.toml

```diff
@@ -1004,7 +1004,7 @@ indent = { tab-width = 2, unit = "  " }
 
 [[grammar]]
 name = "beancount"
-source = { git = "https://github.com/polarmutex/tree-sitter-beancount", rev = "4cbd1f09cd07c1f1fabf867c2cf354f9da53cc4c" }
+source = { git = "https://github.com/polarmutex/tree-sitter-beancount", rev = "f3741a3a68ade59ec894ed84a64673494d2ba8f3" }
```

Comment:
beancount had quite a few changes but it's all refactoring to make the grammar and nodes nicer to work with. It doesn't look like the queries we use need any updates.